src/iop/filmicrgb.c

```diff
@@ -2483,9 +2485,10 @@ int process_cl(struct dt_iop_module_t *self,
   // used to adjust noise level depending on size. Don't amplify noise if magnified > 100%
   const float scale = MAX(piece->iscale / roi_in->scale, 1.f);
 
-  // get the number of OpenCL threads
   uint16_t is_clipped = 0;
-  cl_mem clipped = dt_opencl_alloc_device(devid, 1, 1, sizeof(uint16_t));
+  clipped = dt_opencl_alloc_device_buffer(devid, sizeof(uint16_t));
+  err = dt_opencl_write_buffer_to_device(devid, &is_clipped, clipped, 0, sizeof(uint16_t), CL_TRUE);
```

Comment:
Yes. you initialize the buffer - if i read the diff correctly that **was** missing in the original code!

And - yes again - we had other code parts where this exactly was an issue.